### PR TITLE
correction table visualization

### PR DESCRIPTION
**Pull request by Dari161**
Make a excel sheet, where correction data is visualised with colors.

Also contains some "semi" useless files: a duplicate of everything with dark mode and copyright year auto updater.

### CODE REVIEW

**Comment by Dari161** [comment]
Nincs jobb csv, amin nem gyakrabban vannak az értékek 1 foknál és 1 százaléknál, meg ami kiterjedtebb?

File: assets/sun.png
Comment (NowAlexIndustries) [inline assets/sun.png:1]: This is a placeholder right?

File: base.css
Comment (NowAlexIndustries) [inline base.css:1]: Use pre-existing folder structure before merge-request as this messes up folder structure on origin as well

File: index.js
Comment (NowAlexIndustries) [inline index.js:1]: What if we separated dark and light mode css files and than we would only have to change the associated css file in head? Also lets look into cookies and how to check defaults for a fix for the reload issue

**Review by NowAlexIndustries — CHANGES_REQUESTED** [review]
Please rename 'assets' to 'res' to follow development-wide pre-existing standards.

Also in general dont yeet files into root, and next time create clearly-designated folder for files only meant to aid development.

Before merge, resolve duplicate file issues and general structurelessness in folders.

File: assets/sun.png
Comment (Dari161) [reply assets/sun.png:1]: yep

**Pull request closed without merging**